acceptance/bundle/artifacts/same_name_libraries/output.txt

```diff
@@ -0,0 +1,14 @@
+
+>>> errcode [CLI] bundle deploy
+Building whl1...
+Building whl2...
+Error: Duplicate local library name my_default_python-0.0.1-py3-none-any.whl
```

Comment:
It would be good to see both paths as well here?

Reply:
@denik we show in error message 2 locations in the config where name is used so users can navigate and take a look, I'd rather not to pollute the message with the full paths

Reply:
It's not polluting since it's only shown when there is a problem and in that case any context is valuable. Jumping through yamls is extra work and also yamls have globs so you need to figure out what happened inside CLI (and there could be bugs).

Reply:
Done here https://github.com/databricks/cli/pull/2382/commits/3360d78b90431b7cd31eb47eb84aaee0c3f9e8ca